### PR TITLE
New version: SagerNet.sing-box version 1.8.5

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/137711)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

File: manifests/s/SagerNet/sing-box/1.8.5/SagerNet.sing-box.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+PackageIdentifier: SagerNet.sing-box
+PackageVersion: 1.8.5
+InstallerType: zip
+NestedInstallerType: portable
+ReleaseDate: 2024-02-02
+Installers:
+- Architecture: x86
+  NestedInstallerFiles:
+  - RelativeFilePath: sing-box-1.8.4-windows-386\sing-box.exe
```
Comment (CoolPlayLin) [inline manifests/s/SagerNet/sing-box/1.8.5/SagerNet.sing-box.installer.yaml:12]:
```suggestion
  - RelativeFilePath: sing-box.exe
```

File: manifests/s/SagerNet/sing-box/1.8.5/SagerNet.sing-box.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    PortableCommandAlias: sing-box
+  InstallerUrl: https://github.com/SagerNet/sing-box/releases/download/v1.8.5/sing-box-1.8.5-windows-386.zip
+  InstallerSha256: 5595C2CE3E70E4024D5A29791F8769EB811D6EEF3C8BCB8266259C3E5B28D839
+- Architecture: x64
+  NestedInstallerFiles:
+  - RelativeFilePath: sing-box-1.8.4-windows-amd64\sing-box.exe
```
Comment (CoolPlayLin) [inline manifests/s/SagerNet/sing-box/1.8.5/SagerNet.sing-box.installer.yaml:18]:
```suggestion
  - RelativeFilePath: sing-box.exe
```

File: manifests/s/SagerNet/sing-box/1.8.5/SagerNet.sing-box.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    PortableCommandAlias: sing-box
+  InstallerUrl: https://github.com/SagerNet/sing-box/releases/download/v1.8.5/sing-box-1.8.5-windows-amd64.zip
+  InstallerSha256: 75E089E3742C153D72C214627EFD16AEF4CC493CB48BBD7996BB94EA6F2B05ED
+- Architecture: arm64
+  NestedInstallerFiles:
+  - RelativeFilePath: sing-box-1.8.4-windows-arm64\sing-box.exe
```
Comment (CoolPlayLin) [inline manifests/s/SagerNet/sing-box/1.8.5/SagerNet.sing-box.installer.yaml:24]:
```suggestion
  - RelativeFilePath: sing-box.exe
```

**Review by CoolPlayLin — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Duplicate of https://github.com/microsoft/winget-pkgs/pull/137712

**Pull request closed without merging**